src/main/java/book/online/controller/BookController.java

```diff
@@ -0,0 +1,37 @@
+package book.online.controller;
+
+import book.online.dto.BookDto;
+import book.online.dto.CreateBookRequestDto;
+import book.online.service.BookService;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/books")
+public class BookController {
+    private BookService bookService;
+
+    public BookController(BookService bookService) {
+        this.bookService = bookService;
+    }
+
+    @GetMapping
+    public List<BookDto> getAll() {
```

Comment:
```suggestion
    public List<BookDto> getAllBooks() {
```
try to follow one naming convention